application/app/services/dataverse/hub_registry.rb

```diff
@@ -8,11 +8,12 @@ class HubRegistry
     def initialize(http_client: Common::HttpClient.new(base_url: HUB_API_URL), cache: Rails.cache)
       @http_client = http_client
       @cache = cache
+      @class_name = self.class.name
     end
 
     def installations
       @cache.fetch(CACHE_KEY, expires_in: CACHE_EXPIRY) do
-        log_info('Fetching Dataverse Hub installations...', {url: HUB_API_URL})
+        log_info('Fetching Dataverse Hub installations...', {url: HUB_API_URL}, @class_name)
```

Comment:
not sure why we need to explicitly pass to the logger the @class_name if internally the logger is assigned by default to `self.class.name`

Reply:
ok. maybe there is another metaprogramming method that returns the class where the module is included but it is not worth investing much time on it I guess